xdsl/ir.py

```diff
@@ -184,8 +184,8 @@ def erase(self, safe_erase: bool = True) -> None:
         """
         if safe_erase and len(self.uses) != 0:
             raise Exception(
-                "Attempting to delete SSA value that still has uses of operation:\n"
-                f"{self.op}")
+                "Attempting to delete SSA value that still has uses of result "
+                f"of operation:\n{self.owner}")
```

Comment:
This could be a BlockArg as well, right? I know you are doing this for typing, so I am fine with leaving this as is and changing it later on, just noticed :)

Reply:
Yeah I was wondering about this, seems like a problem for later :)